src/data-dictionary/events/NrAiIncident/tags.md

```diff
@@ -0,0 +1,10 @@
+---
+name: tags.*
+type: attribute
+events:
+  - NrAiIncident
+---
+
+Arbitrary key-value metadata, or tags, associated with the violation. `tag.` is the prefix and `*` is the metadata/tag name. For details on how to use this, see the documentation for [muting rules](/docs/alerts/new-relic-alerts/managing-notification-channels/muting-rules-suppress-notifications) or [description](/docs/alerts/new-relic-alerts/defining-conditions/alert-condition-descriptions).
+
```

Comment:
I'd specify what is the description about, it looks out of context.

Reply:
Good call!